src/index.js

```diff
@@ -366,6 +428,8 @@ class CreditCardInput extends Component<Props, State> {
       inputStyle,
       invalidStyle
     } = this.props;
+    const transformedStyle = showZip ? {transform: 'translateX(0)'} : {transform: 'translateX(4rem)'}
```

Comment:
Might be better to move this into the `InputWrapper` styled component and pass `showZip` as a prop to it?

E.g.

```javascript
const InputWrapper = styled.label`
  ...
  transform: translateX(${props => props.showZip ? '0' : '4rem' });
  ...
`;

<InputWrapper
  inputStyled={inputStyle}
  data-max="MM / YY 9"
  showZip={showZip}
>
```